platform/administer/templates/overview.mdx

```diff
@@ -0,0 +1,820 @@
+    default: false
+  
+  # Database configuration only required when database is enabled
+  - name: database_size
+    type: string
+    required: true
+    condition: "{{ .Values.enable_database }}"
+    validation:
+      enum: ["small", "medium", "large"]
+  
+  - name: database_backup_schedule
+    type: string
+    default: "0 2 * * *"
+    condition: "{{ .Values.enable_database }}"
+    validation:
+      pattern: "^[0-9*/ ,-]+$"
+      patternDescription: "Valid cron expression"
+```
+
+## Template validation and schema enforcement
+
+Templates support complete JSON Schema validation to ensure parameter values meet your requirements and to prevent configuration errors before cluster creation.
+
+### Schema validation structure
+
+JSON Schema validation operates at multiple levels:
+
+```yaml
+apiVersion: vcluster.loft.sh/v1
+kind: VClusterTemplate
+metadata:
+  name: enterprise-template
+spec:
+  # Parameter-level validation (shown above)
+  parameters:
+    - name: team_name
+      type: string
+      validation:
+        pattern: "^(frontend|backend|platform|security)$"
+  
+  # Template-level validation schema
+  jsonSchema:
+    type: object
+    required: ["team_name", "cost_center"]
+    properties:
+      team_name:
+        type: string
+        description: "Team responsible for this cluster"
+        enum: ["frontend", "backend", "platform", "security"]
+      
+      cost_center:
+        type: string
+        description: "Billing cost center code"
+        pattern: "^CC-[0-9]{4}$"
+        examples: ["CC-1001", "CC-2003"]
+      
+      resource_tier:
+        type: string
+        description: "Resource allocation tier"
+        enum: ["small", "medium", "large", "xlarge"]
+        default: "medium"
+      
+      # Complex validation for resource requests
+      custom_resources:
+        type: object
+        properties:
+          cpu:
+            type: string
+            pattern: "^[0-9]+m$"
+            description: "CPU in millicores"
+          memory:
+            type: string  
+            pattern: "^[0-9]+[GMK]i$"
+            description: "Memory with unit (Gi, Mi, Ki)"
+        additionalProperties: false
+        dependencies:
+          cpu: ["memory"]
+          memory: ["cpu"]
+    
+    # Conditional validation based on environment
+    if:
+      properties:
+        environment:
+          const: "production"
+    then:
+      required: ["backup_policy", "monitoring_config"]
+      properties:
+        backup_policy:
+          type: object
+          required: ["enabled", "retention_days"]
+        monitoring_config:
+          type: object
+          required: ["alerts_enabled", "dashboard_enabled"]
+```
+
+### Custom validation functions
+
+For complex business logic validation, you can define custom validation functions:
+
+```yaml
+spec:
+  parameters:
+    - name: cluster_size
+      type: string
+      validation:
+        # Custom validation function
+        customValidation: |
```

Comment:
I don't think this exists. There's support for `validation` and `invalidation` regexes.